README.md

```diff
@@ -48,6 +48,8 @@ DATABASE_USER=postgres
 DATABASE_PASSWORD=postgres
 
 SECRET=asdlplplfwfwefwekwself.2342.dawasdq
+
+MongoUri=mongodb://localhost:27017/mydatabase
```

Comment:
Suggestion: MONGO_URI instead of MongoUri

Reply:
yes. changed to MONGO_URI